tests/is/L42/perftests/PerfMonitor.java

```diff
@@ -0,0 +1,35 @@
+package is.L42.perftests;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import is.L42.top.CachedTop;
+
+public class PerfMonitor {
+	private final MemoryMonitor mem;
```

Comment:
Some files seams to be formatted with tabs, or at least, with much more than 2 spaces